packages/api/lib/testUtils.js

```diff
@@ -161,6 +161,8 @@ function fakePdrFactoryV2(params = {}) {
     provider: 'fakeProvider',
     status: 'completed',
     createdAt: Date.now(),
+    progress: 0,
+    execution: 'execution',
```

Comment:
Adding this is blowing up the PDR endpoint test as it's trying to translate a record with this execution default in it.